database/database.go

```diff
@@ -32,6 +32,17 @@ type Handler struct {
 	// Implement methods if necessary
 }
 
+func (h *Handler) DBClose() error {
```

Comment:
Is it reasonable that this function be invoked when a SIGTERM is trapped?

Reply:
I think yes, it is reasonable. SInce the default time betweent first SIGTERM and SIGKILL is 30 seconds in kubernetes, that is more than enough to close the connections if the pod terminates. Same with docker stop.

Reply:
And for Docker, it's 10 seconds, I believe.

Good. Please make it so.

Reply:
After this is merged, when I will create the PR in Meshery I will add SIGTERM as one of the signals on which the process will trap.

Reply:
Perfect. Thank you, @Revolyssup

Reply:
And add this function that trap?